src/use_case/login/LoginDataAccessInterface.java

```diff
@@ -3,7 +3,7 @@
 import entity.Project;
 
 public interface LoginDataAccessInterface {
-    boolean projectExist(String identifier);
+    boolean existsByName(String identifier);
```

Comment:
I think you need to edit the FirebaseDAO for this to actually work

Reply:
For existsByName(), I am not sure whether it works with our current save(). My idea for this is that we create a separate collection in the database, let's say "Project Name Collection", and we directly store all the project names in a document in this collection. So that for existsByName(), we directly access this collection. The reason for this collection needed is that I am not sure how to go through each collection without the collection name known. If you think this works, I'll edit the FirebaseDAO this way.

Reply:
We haven't figured out a way to search through our firebaseDAO yet, I think Kosei will work on it this week and we can start from there